<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1074</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>595</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="minimumSize">
             <size>
              <width>87</width>
              <height>26</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>87</width>
              <height>26</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;设备IP&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_ipaddr">
            <property name="minimumSize">
             <size>
              <width>127</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>127</width>
              <height>22</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="minimumSize">
             <size>
              <width>87</width>
              <height>26</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>87</width>
              <height>26</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;用户名&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_username">
            <property name="minimumSize">
             <size>
              <width>127</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>127</width>
              <height>22</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="minimumSize">
             <size>
              <width>87</width>
              <height>26</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>87</width>
              <height>26</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;密码&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_passwd">
            <property name="minimumSize">
             <size>
              <width>127</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>127</width>
              <height>22</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="minimumSize">
             <size>
              <width>83</width>
              <height>149</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>83</width>
              <height>149</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;命令&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_import">
            <property name="minimumSize">
             <size>
              <width>83</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>83</width>
              <height>32</height>
             </size>
            </property>
            <property name="text">
             <string>批量导入</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="te_cmd_input">
          <property name="minimumSize">
           <size>
            <width>256</width>
            <height>192</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>256</width>
            <height>192</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>98</width>
          <height>29</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>98</width>
          <height>29</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;当前运行结果&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="te_cmd_output">
        <property name="styleSheet">
         <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="btn_conn_test">
          <property name="minimumSize">
           <size>
            <width>137</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>137</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>连接性测试</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_export">
          <property name="minimumSize">
           <size>
            <width>136</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>136</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>导出报告</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_inspection">
          <property name="minimumSize">
           <size>
            <width>137</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>137</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>巡检</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_about">
          <property name="minimumSize">
           <size>
            <width>137</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>137</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>关于</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
